app/views/layouts/application.html.erb

```diff
@@ -5,32 +5,37 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <%= stylesheet_link_tag 'application', media: 'all' %>
   <%= csrf_meta_tags %>
+
+  <link href="https://fonts.googleapis.com/css2?family=Orbitron:wght@600&display=swap" rel="stylesheet">
+
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/foundation-sites@6.4.3/dist/css/foundation.min.css" integrity="sha256-GSio8qamaXapM8Fq9JYdGNTvk/dgs+cMLgPeevOYEx0= sha384-wAweiGTn38CY2DSwAaEffed6iMeflc0FMiuptanbN4J+ib+342gKGpvYRWubPd/+ sha512-QHEb6jOC8SaGTmYmGU19u2FhIfeG+t/hSacIWPpDzOp5yygnthL3JwnilM7LM1dOAbJv62R+/FICfsrKUqv4Gg==" crossorigin="anonymous">
+  <link rel="stylesheet" href="public/stylesheets/style.css">
 </head>
+
 <body>
   
   <nav class="top-bar">
     <h1><%= link_to "Quantum Complainers: A Sci-Fi Movie Review Site", root_path%></h1>
 
     <section class="top-bar-section">
-      <ul class="right">
+      <div class="right">
         <%- if current_user -%>
-          <li><%= link_to 'Sign Out', destroy_user_session_path, method: :delete %></li>
+          <p><%= link_to 'Sign Out', destroy_user_session_path, method: :delete %></p>
         <%- else -%>
-          <li><%= link_to 'Sign Up', new_user_registration_path %></li>
-          <li><%= link_to 'Sign In', new_user_session_path %></li>
+          <p><%= link_to 'Sign Up', new_user_registration_path %></p>
+          <p><%= link_to 'Sign In', new_user_session_path %></p>
         <%- end -%>
-      </ul>
+      </div>
     </section>
   </nav>
 
-  <%- flash.each do |key, value| -%>
-    <div class="flash flash-<%= key %>">
-      <%= value %>
-    </div>
-  <%- end -%>
+  <div class="body-container"> 
+  <div class="grid-container">
```

Comment:
Indentation should be updated here for readability

Reply:
Indentation updated